src/view/domconverter.js

```diff
@@ -278,6 +278,12 @@ export default class DomConverter {
 				domAfter = domParent.childNodes[ 0 ];
 			} else {
 				domBefore = this.getCorrespondingDom( viewPosition.nodeBefore );
+
+				if ( !domBefore ) {
```

Comment:
> I'm generally ok with this, because the only alternative is that viewPositionToDom will be called in a try-catch which smells really bad.

Yes, most of the view DOM conversion methods can return null, to avoid throwing. Also, I agree with you that methods, which use then, should be prepared for such scenario. But note that not in all cases such null is possible. For instance, in if you just rendered content you should be able to find a corresponding element/position without expecting a null. So we should check if the function return null in many cases, but not in all of them.